classes/Entities/ACL/UserRole.php

```diff
@@ -14,4 +14,9 @@ class UserRole extends BaseModel
      * @var string
      */
     protected $table = 'acl_user_roles';
+
+    /**
+     * @var string[]
+     */
+    protected $fillable = ['user_id'];
```

Comment:
Actually, instead can we flip this to be `protected $guarded = ['id'];`. is simpler that way as we'd really always want to be able to add data to this table only blocking 'id'